index.js

```diff
@@ -11,16 +14,23 @@ exports.handler = async (event) => {
     const query = event.queryStringParameters || {}
     const headers = util.keysToLowerCase(event.headers) || {}
     const identity = (event.requestContext || {}).identity || {}
+    const userId = cookie.parse(headers['cookie'] || '')[COOKIE_NAME]
 
     if (event.httpMethod !== 'GET' && event.httpMethod !== 'HEAD') {
       return util.text(405, 'Method not allowed')
     } else if (event.path === '' || event.path === '/') {
       return pages.home()
     } else if (event.path === '/admin') {
       return await pages.admin(query, headers)
-    } else if (event.path === '/i.gif') {
-      await tracker.log(query, headers, identity)
+    } else if ((event.path === '/i.gif' && userId) || event.path === '/r.gif') {
```

Comment:
Should this be `r.gif && userId` here? Just thinking: we start on i and redirect to r, maybe having a cookie. Not sure how we get back to i w/ a cookie.

Reply:
This would be a returning user, that already had a cookie set.  Could 302 them, but thought I'd save them a redirect leg and just log on the 1st request.

Reply:
+1